www/README.md

```diff
@@ -0,0 +1,20 @@
+## Effection Website
+
+The Effection website contains Guides and API documentation.
```

Comment:
We should include a phrase about how it introspects the Effection repository at various branches and tags as the source of documentation.